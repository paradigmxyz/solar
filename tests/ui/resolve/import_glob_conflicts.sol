import "./aux/udvt.sol";
import * from "./aux/udvt2.sol"; //~ ERROR: already declared
