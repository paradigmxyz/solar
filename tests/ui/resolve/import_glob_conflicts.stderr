error: identifier `MyUdvt` already declared
  --> ROOT/tests/ui/resolve/import_glob_conflicts.sol:LL:CC
   |
LL | import "./aux/udvt.sol";
   | ------------------------ note: previous declaration imported here
LL | import * from "./aux/udvt2.sol";
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
  ::: ROOT/tests/ui/resolve/aux/udvt.sol:LL:CC
   |
LL | type MyUdvt is uint256;
   | ----------------------- note: previous declaration declared here
   |

error: aborting due to 1 previous error

